portfolio/src/main/webapp/style.css

```diff
@@ -40,13 +51,63 @@ cursive {
 }
 
 div.gallery img {
-  height: auto;
+  height: 75%;
   width: 100%;
 }
 
 div.desc {
+  background-color: white;
+  font-family: cursive;
+  font-size: 14px;
+  margin-top: -4px;
+  padding: 13px;
+  text-align: center;
+  width: 177px;
+}
+
+media-title {
   font-family: cursive;
-  font-size: 12px;
-  padding: 2px;
+  margin-top: 10px;
+} 
+
+.fa {
+  border-radius: 50%;
+  cursor: pointer;
```

Comment:
Links will already have the pointer cursor, so you should be able to remove this.